infinitest-eclipse/src/main/java/org/infinitest/eclipse/prefs/PreferencesConstants.java

```diff
@@ -41,4 +41,8 @@
 
 	public static final String SLOW_TEST_WARNING = "org.infinitest.eclipse.slow-warning";
 
+	public static final String FAIL_BACKGROUND_COLOR = "org.infinitest.eclipse.color.fail.background";
```

Comment:
To be consistent with the naming of `setFailingColors` the constants `FAIL_BACKGROUND_COLOR` and `FAIL_TEXT_COLOR` should be renamed `FAILING_BACKGROUND_COLOR` and `FAILING_TEXT_COLOR`